bats/tests/containers/factory-reset.bats

```diff
@@ -159,17 +159,15 @@ check_directories() {
     delete_dir=("$PATH_CONFIG")
     if is_unix; then
         delete_dir+=("$HOME/.rd")
+        delete_dir+=("$PATH_APP_HOME/credential-server.json" "$PATH_APP_HOME/lima" "$PATH_APP_HOME/rd-engine.json" "$PATH_LOGS")
```

Comment:
FWIW, we can change `"$PATH_APP_HOME/lima` to `"$LIMA_HOME"`.

Also, unrelated: what about `PATH_EXTENSIONS`?

Reply:
`PATH_EXTENSIONS` deleting needs to go into a separate issue. I'm already stretching this a bit for PRs, like replacing calls to `os.UserConfigDir()` with our own `paths` module (which is def better).

Reply:
Can you create an issue for dealing with `$PATH_EXTENSIONS`?

Reply:
Done in #5599